python/caffe/test/test_draw.py

```diff
@@ -1,7 +1,7 @@
 import os
 import unittest
 
-from google import protobuf
+from google.protobuf import text_format
```

Comment:
The existing syntax is more clear, but its returning error:
`AttributeError: 'module' object has no attribute 'text_format'`

Replacing the import statement as mentioned in PR fixed it. `draw_net.py` also uses the same syntax, probably because of this issue.